packages/media-sound/cava/cava-0.8.1.exheres-0

```diff
@@ -27,11 +28,23 @@ DEPENDENCIES="
         ncurses?        ( sys-libs/ncurses )
         pulseaudio?     ( media-sound/pulseaudio )
         portaudio?      ( media-libs/portaudio )
+        sdl?            ( media-libs/SDL:2 )
         sndio?          ( sys-sound/sndio )
 "
 
 DEFAULT_SRC_CONFIGURE_OPTION_ENABLES=(
-    "alsa input-alsa" "iniparser system-iniparser" "ncurses output-ncurses"
-    "pulseaudio input-pulse" "portaudio input-portaudio" "sndio input-sndio"
+    "alsa input-alsa"
+    "ncurses output-ncurses"
+    "pulseaudio input-pulse"
+    "portaudio input-portaudio"
+    "sdl output-sdl"
+    "sndio input-sndio"
 )
 
+src_prepare() {
+    # External commands essential to compilation in autogen.sh
+    edo echo ${PV} > version
```

Comment:
Since the `echo` would never fail but the actual write to the file can, we prefer this way:
```
edo tee version <<CONTENTS
${PV}
CONTENTS